src/App/utils/auth.js

```diff
@@ -0,0 +1,57 @@
+import { verify, sign} from 'jsonwebtoken';
+import { config } from 'dotenv';
+import errorRes from './errHandler';
+import { hash, compare } from 'bcryptjs';
+
+config();
+
+const authenticateToken = async (req, res, next) => {
```

Comment:
This has been done like 3 times now in your API!
This is totally similar to `/middlewares/token.js` and `/middlewares/protect.js` Why do you need it three times?